# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-glibc-2.43.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-144.0.7559.109/sandbox/linux/system_headers/linux_seccomp.h.vanilla	2026-01-28 20:12:45.878244938 +0100
+++ chromium-144.0.7559.109/sandbox/linux/system_headers/linux_seccomp.h	2026-01-28 20:12:55.307004818 +0100
@@ -215,7 +215,7 @@
 #endif
 
 #ifndef SYS_SECCOMP
-#define SYS_SECCOMP                   1
+//#define SYS_SECCOMP                   1
 #endif
 
 #endif  // SANDBOX_LINUX_SYSTEM_HEADERS_LINUX_SECCOMP_H_
